### DISCUSSION TITLE
Mermaid Mobile Support

### DISCUSSION THREAD

**Discussion by askpt** [community/community#11595] [Mobile] [answered]
Hello all!

Mermaid support has been announced for the markdown pages but when I try open on my phone it still renders as a code instead of the diagram itself.

Are you planning to integrate mermaid also on the mobile version of GitHub?

See screenshot:
![image](https://user-images.githubusercontent.com/2493377/154238757-86e8fcf5-cc1b-4524-baa2-33fd0dd4238e.jpeg)

**Comment by brianlovin** [comment, marked as answer]
Thanks for flagging this @askpt — we don't have this scheduled on our roadmap right now, but we are tracking this to make sure we keep up with all the new features GitHub is adding to markdown rendering.

**Reply by askpt** [reply]
Thanks, @brianlovin for the feedback! Cannot wait to have it available on the mobile app!

**Reply by sidedwards** [reply]
@brianlovin is there a reason for not including mermaid mobile support? We heavily use mermaid diagrams across many organizations for software diagrams and having them render on mobile would be very helpful.

**Reply by alphatownsman** [reply]
Mobile app is good for reading doc, not supporting mermaid rendering is a big gap.

**Reply by rinczefi-user** [reply]
Hi @brianlovin 👋 
Is there any update on this one?
Thanks!

**Reply by PLukas2018** [reply]
@brianlovin Would it be possible to render at least some placeholder image instead of showing the code block?

**Reply by phizlip** [reply]
Has there been any update in development or the roadmap for this feature? @brianlovin

**Comment by Permal1978** [comment]
I am whery gratefull for all the help i can Get. And willing to tet everything.

**Comment by Boneheinwai6801** [comment]
render on mobile would be very helpful.

**Comment by DAVIDKIMGANGMIN** [comment]
To print emojis in C# and VB.NET, you can use the `Console.WriteLine()` method. Here's an example in C#:

```csharp
using System;

class Program
{
    static void Main()
    {
        Console.WriteLine("🖨");
    }
}
```

This program will print the printer emoji "🖨" to the console when run.

Please note that emojis may not display correctly in all console environments, as some consoles may not support Unicode characters. If you encounter any issues with emojis not displaying properly, you may need to check your console settings or use a different console application that supports Unicode.
